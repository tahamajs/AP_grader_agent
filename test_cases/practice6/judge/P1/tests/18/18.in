POST signup ? username "mehdi" password "mn1234"
POST login ? username "mehdi" password "mn1234"
POST reserve ? restaurant_name "shirazo" table_id "2" start_time "8" end_time "10" foods "burger"
POST reserve ? restaurant_name "shiraz" table_id "10" start_time "8" end_time "10" foods "burger"
POST reserve ? restaurant_name "shiraz" table_id "3" start_time "8" end_time "10" foods "felafel"
POST reserve ? restaurant_name "shiraz" table_id "2" start_time "8" end_time "25" foods "burger"
POST reserve ? restaurant_name "shiraz" table_id "2" start_time "2" end_time "5"
POST reserve ? restaurant_name "shiraz" table_id "2" start_time "8" end_time "10" foods "burger"
POST reserve ? restaurant_name "eram" table_id "2" start_time "9" end_time "11" foods "joojeh"
POST reserve ? restaurant_name "shiraz" table_id "2" start_time "7" end_time "9"