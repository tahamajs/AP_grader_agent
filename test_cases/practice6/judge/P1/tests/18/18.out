OK
Permission Denied
Not Found
Not Found
Not Found
Permission Denied
Permission Denied
Reserve ID: 1
Table 2 for 8 to 10 in shiraz
Price: 250
Permission Denied
Permission Denied